develop/toolkit/api-libraries/py-substrate-interface.md

```diff
@@ -1,4 +1,81 @@
 ---
 title: Python Substrate Interface
-description: TODO
----
\ No newline at end of file
+description: Learn how to connect to Polkadot SDK-based nodes, query data, submit transactions, and manage blockchain interactions using the Python Substrate Interface.
+---
+
+# Python Substrate Interface
+
+## Introduction
+
+This Python library specializes in interfacing with a Polkadot SDK-based node, querying storage, composing extrinsics, SCALE encoding/decoding, and providing additional convenience methods to deal with the features and metadata of the Substrate runtime.
+
+The [Python Substrate Interface](https://github.com/polkascan/py-substrate-interface){target=\_blank} is a powerful library that enables interaction with Polkadot SDK-based chains. It provides essential functionality for:
+
+- Querying on-chain storage
+- Composing and submitting extrinsics
+- SCALE encoding/decoding
+- Interacting with Substrate runtime metadata
+- Managing blockchain interactions through convenient utility methods
```

Comment:
This intro feels a bit repetitive. For example, you mention in the first sentence querying storage, composing extrinsics, SCALE encoding/decoding, convenience methods and then all of these items are repeated in the list.

Maybe just delete the first paragraph? 🤷‍♀️